src/Message/AbstractRequest.php

```diff
@@ -1,217 +1,217 @@
 <?php
 
+declare(strict_types=1);
+
 namespace Omnipay\WestpacPaywayRest\Message;
 
+use Money\Money;
+use Omnipay\Common\Message\ResponseInterface;
+
 /**
  * @link https://www.payway.com.au/rest-docs/index.html
  */
 abstract class AbstractRequest extends \Omnipay\Common\Message\AbstractRequest
 {
-    /** @var string Endpoint URL */
-    protected $endpoint = 'https://api.payway.com.au/rest/v1';
+    protected string $endpoint = 'https://api.payway.com.au/rest/v1';
```

Comment:
Wonder if this would be better as a constant 🤔

Reply:
Updated